modules/microshift-embed-microshift-image-offline-deploy.adoc

```diff
@@ -0,0 +1,113 @@
+
+.Prerequisites
+
+* You have root-user access to your build host.
+* Your build host meets the Image Builder system requirements.
+* You have installed and set up Image Builder and the composer-cli tool.
+* You have created a {op-system-ostree} image blueprint.
+* You have installed jq.
+
+.Procedure
+
+. Get the exact list of container image references used by the {microshift-short} version you are deploying. You can either install the `microshift-release-info` RPM package by following step 2 or download and unpack the RPM by following step 3.
+
+. To install the `microshift-release-info` RPM package:
+
+.. Install the `microshift-release-info` RPM package by running the following command:
++
+[source,terminal]
+----
+$ sudo dnf install -y microshift-release-info-<release_version>
+----
+Replace ,`<release_version>` with the numerical value of the release you are deploying, using the entire version number, such as `4.14.0`.
+
+.. List the contents of the `/usr/share/microshift/release` directory to verify the presence of the release information files by running the following command:
++
+[source,terminal]
+----
+$ ls /usr/share/microshift/release
+----
++
+.Example output
+[source,terminal]
+----
+release-x86_64.json
+----
++
+If you installed the `microshift-release-info` RPM, you can proceed to step 4.
+
+. If you did not complete step 2, download and unpack the `microshift-release-info` RPM without installing it:
+
+.. Download the RPM package by running the following command:
++
+[source,terminal]
+----
+$ sudo dnf download microshift-release-info-<release_version>
+----
+Replace ,`<release_version>` with the numerical value of the release you are deploying, using the entire version number, such as `4.14.0`.
++
+.Example rpm
+[source,terminal]
+----
+microshift-release-info-4.14.0.*.el9.noarch.rpm <1>
+----
+<1> The `*` represents the date and commit ID. Your output should contain both, for example `-202311101230.p0.g7dc6a00.assembly.4.14.0`.
+
+.. Unpack the RPM package without installing it by running the following command:
++
+[source,terminal]
+----
+$ rpm2cpio <my_microshift_release_info_4.14.0.*.el9.noarch.rpm> | cpio -idmv <1>
+./usr/share/microshift/release/release-aarch64.json
+./usr/share/microshift/release/release-x86_64.json
+----
+<1> Replace `<my_microshift_release_info_4.14.0.*.el9.noarch.rpm>` with the name of the RPM package from the previous step.
+
+. Using the release information for your architecture, generate the section to embed the container images to your blueprint TOML file:
++
+[source,terminal]
+----
+$ jq -r '.images | .[] | ("[[containers]]\nsource = \"" + . + "\"\n")' release-$(uname -m).json >> <my_blueprint.toml>
+----
+Replace <my_blueprint.toml> with the name of the blueprint file that you are modifying.
+
+. Pin the version of the {microshift-short} RPMs in the blueprint to the version matching your container images. See the following  example section to add to a TOML:
```

Comment:
Starting from here and until the end, the section need to be rewritten. 
Note that the command on line 79 already appends the list of container images to the blueprint. So, this explanatrion makes an impression that more changes need to be made.